basyx.conceptdescriptionrepository/basyx.conceptdescriptionrepository-core/src/test/java/org/eclipse/digitaltwin/basyx/conceptdescriptionrepository/core/ConceptDescriptionRepositorySuite.java

```diff
@@ -259,7 +260,7 @@ private void assertConceptDescriptionsAreContained(Collection<ConceptDescription
 		assertTrue(actualConceptDescriptions.containsAll(expectedConceptDescriptions));
 	}
 
-	private ConceptDescription createDummyConceptDescription(String id) {
+	protected ConceptDescription createDummyConceptDescription(String id) {
```

Comment:
When the test cases are moved in this class (as suggested above) then this change is not required.

Reply:
done